docs/sources/developers/plugins/cross-plugin-linking.md

```diff
@@ -1,21 +1,23 @@
 ---
-title: Cross plugin links
+title: Work with cross-plugin links
 description: Learn how to add plugin links to a Grafana app plugin
 ---
 
-# Cross plugin links
+# Work with cross-plugin links
 
-Using the Plugin extensions API, App plugins can register extension points of their own to display other plugins links. This cross-plugin linking creates a more immersive user experience for installed plugins.
+With the [plugin API](https://grafana.com/docs/grafana/latest/developers/plugins/backend/grafana-plugin-sdk-for-go/), app plugins can register extension points of their own to display other plugins links. This is called _cross-plugin linking_, and you can use it to create more immersive user experiences with installed plugins.
```

Comment:
This is incorrect and the link should be removed. There is no need to use the go sdk for plugin extensions. We just call it "Plugin Extensions API". Here we are talking about basically a Java Script method call that a plugin author has to add to their App plugins React code to add a new "Plugin Extension Point" that other Plugins can use.

Reply:
Similar comment above

Reply:
I've changed this to "Plugins extension API" (not linked). On a minor style note, we don't use Title Case if we can avoid it at all. Regarding "Plugins extension API",  I hesitate to throw out a reference like this without explaining it because we can't assume that the user will know what this is. 

Would it be safe to add this sentence?: "By Plugins extension API, we mean a JavaScript method call that you have to add to your App lugin's ReactJS code to create a new Plugin extension point that other plugins can use." 

Or, if there's a doc where we talk about this concept in more detail, then we could add a link to it.

Reply:
removed, thanks